apps/fulfillment-e2e/src/support/commands.ts

```diff
@@ -4,6 +4,7 @@ declare global {
   namespace Cypress {
     interface Chainable {
       clearIndexedDB(): void;
+      findByTestId(selector: string): Chainable<JQuery<HTMLElement>>;
```

Comment:
Do we need the `getByTestId` command as well? (or we can add it later?)

Reply:
Let's remove the `findByTestId` for now